tgui/yarn.lock

```diff
@@ -1951,6 +1951,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@popperjs/core@npm:^2.9.1":
+  version: 2.9.3
```

Comment:
tgstation locked to 2.9.2 because I guess that was what was available at the time. I'm fine with ours being ahead by a patch version.